pixi/cloud-function/package.json

```diff
@@ -10,6 +10,6 @@
 	"author": "AMPHTML Authors",
 	"license": "Apache 2",
 	"dependencies": {
-		"@ampproject/toolbox-page-experience": "2.8.0-canary.8"
+		"@ampproject/toolbox-page-experience": "2.8.0-canary.11"
```

Comment:
any reason we are using 11 here and [9 in the main package](https://github.com/ampproject/amp.dev/pull/5561/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R63)?

Reply:
optimizer is not yet on 11

Reply:
oh pft. misread. apologizes.